### DISCUSSION TITLE
No entiendo como funcion GitHub...

### DISCUSSION THREAD

**Discussion by harpomaxx** [harpomaxx/algoritmos2#3] [General]
La verdad que es muy complicado.. nose por que no usamos wasup.

**Comment by ExeTcod** [comment]
nosotros tampoco

**Reply by harpomaxx** [reply]
para lidiar con la depresión les dejo el link a las slides de github :face_with_head_bandage:

**Comment by PaulaMartinez48** [comment]
podemos usar discord :D

**Reply by harpomaxx** [reply]
En IA 1 usamos discord. De hecho fue una opcion, pero la verdad que seamos sinceros.. no escriben tanto como para que valga la pena tener abierto un canal mas jaja. En cambio github y su app mobile, la tengo siempre activa.
